hyperstart.go

```diff
@@ -305,7 +323,9 @@ func (h *hyper) bindUnmountContainerRootfs(podID, cID string) error {
 
 func (h *hyper) bindUnmountAllRootfs(pod Pod) {
 	for _, c := range pod.containers {
+		h.bindUnmountContainerMounts(pod.id, c)
 		h.bindUnmountContainerRootfs(pod.id, c.id)
+		h.bindUnmountContainerMounts(pod.id, c)
```

Comment:
One is for the container rootfs and the other is for the bind-mounts that we created in the shared directory